### PR TITLE
Updates for v19.9

### PR DESCRIPTION
**Pull request by felipe-barajas**
Updated documentation for 19.9 release

### CODE REVIEW

File: pages/open_cas_linux/guide_configuring.md [hunk 1]
```diff
@@ -431,4 +434,9 @@ that exported devices have been configured to agree with the contents of the
 
 >>   ...  
 >>   /dev/cas1-1p1 /mnt/cache1 xfs defaults 0 0  
->>   /dev/cas1-2p1 /mnt/cache2 xfs defaults 0 0
+>>   /dev/cas1-2p1 /mnt/cache2 xfs defaults 0 0  
+
+
+*NOTE:* In the case when the caching device if formatted with a 4KiB physical block size
```
Comment (KamilLepek) [inline pages/open_cas_linux/guide_configuring.md]: ~~```if```~~``` is formatted```

File: pages/open_cas_linux/guide_tool_details.md [hunk 2]
```diff
@@ -817,53 +817,86 @@ alru flushing policy to be used for the applicable cache instance.
...
+
+**[-p, -\-policy \<POLICY\>]:** The policy desired for promotion of core lines  
+
+*Promotion policies:*
+
+-   *always* - Core lines are attemped to be promoted each time they are accessed.
+
+-   *nhit* - Core lines are attemped to be promoted after n-th access.
+
+When the policy is *nhit* set the n-th access using one of these available commands:
```
Comment (KamilLepek) [inline pages/open_cas_linux/guide_tool_details.md]: I don't get the sentence above.

File: pages/open_cas_linux/guide_tool_details.md [hunk 3]
```diff
@@ -930,29 +963,63 @@ Time and Flush Max Buffers.
...
+**Usage:** casadm –\-get-param –\-name promotion –\-cache-id \<CACHE_ID\>
+
+**Example:**
+
+>   \# casadm -\-get-param -\-name promotion -\-cache-id 1 -\-output-format csv  
+
+or
+>   \# casadm –G –n promotion –i 1
+
+**Description:** Retrives the promotion policy
```
Comment (KamilLepek) [inline pages/open_cas_linux/guide_tool_details.md]: ```Retrieves```

File: pages/open_cas_linux/guide_tool_details.md [hunk 4]
```diff
@@ -930,29 +963,63 @@ Time and Flush Max Buffers.
...
+**Usage:** casadm –\-get-param –\-name promotion-nhit –\-cache-id \<CACHE_ID\>
+
+**Example:**
+
+>   \# casadm -\-get-param -\-name promotion-nhit -\-cache-id 1 -\-output-format csv  
+
+or
+>   \# casadm –G –n promotion-nhit –i 1
+
+**Description:** Retrives the nhit promotion policy values
```
Comment (KamilLepek) [inline pages/open_cas_linux/guide_tool_details.md]: ```Retrieves```

File: pages/open_cas_linux/guide_configuring.md (hunk 1 above)
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_configuring.md]: Fixed in commit 2246e9b

File: pages/open_cas_linux/guide_tool_details.md (hunk 3 above)
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_tool_details.md]: Fixed in commit 2246e9b

File: pages/open_cas_linux/guide_tool_details.md [hunk 5]
```diff
@@ -721,26 +721,26 @@ reads or writes.
...
-**[-j, --core-id \<ID\>]**: Unique identifier for core \<0 to 4095\>. If not
+**[-j, -\-core-id \<ID\>]**: Unique identifier for core \<0 to 4095\>. If not
 specified, core parameter is set to all cores in a given cache.
 
 ###  cleaning
 
-**NOTE:** Because the --name cleaning options provide improved functionality
+**NOTE:** Because the -\-name cleaning options provide improved functionality
 over the -D/--flush-parameters command, -D/--flush-parameters is now obsolete
 and will be removed in a future release.
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_tool_details.md]: Fixed in commit ae8644e

File: pages/open_cas_linux/guide_tool_details.md [hunk 6]
```diff
@@ -817,53 +817,85 @@ alru flushing policy to be used for the applicable cache instance.
...
+
+**Example:**  
+
+>   \# casadm -\-set-param -\-name promotion -\-cache-id 1 -\-policy always
+
+or  
+>   \# casadm -X -n promotion -i 1 -p always
+
+
+**Description:** This parameter specifies the desired promotion of core lines
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_tool_details.md]: Fixed in commit ae8644e

File: pages/open_cas_linux/guide_tool_details.md [hunk 7]
```diff
@@ -817,53 +817,85 @@ alru flushing policy to be used for the applicable cache instance.
...
+**[-p, -\-policy \<POLICY\>]:** The policy desired for promotion of core lines  
+
+*Promotion policies:*
+
+-   *always* - Core lines are attemped to be promoted each time they are accessed.
+
+-   *nhit* - Core lines are attemped to be promoted after the n-th access.
+             This n-th access threshold can be set using one of these commands:
+
+       casadm –set-param –name promotion-nhit -\-cache-id \<CACHE_ID\> [-o, -\-trigger] \<PERCENTAGE\>
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_tool_details.md]: Fixed in commit ae8644e

File: pages/open_cas_linux/guide_tool_details.md [hunk 8]
```diff
@@ -817,53 +817,85 @@ alru flushing policy to be used for the applicable cache instance.
...
+
+-   *always* - Core lines are attemped to be promoted each time they are accessed.
+
+-   *nhit* - Core lines are attemped to be promoted after the n-th access.
+             This n-th access threshold can be set using one of these commands:
+
+       casadm –set-param –name promotion-nhit -\-cache-id \<CACHE_ID\> [-o, -\-trigger] \<PERCENTAGE\>  
+       **Description:** Percent of cache to be occupied before cache inserts will be filtered by the policy.  
+
+       casadm –set-param –name promotion-nhit -\-cache-id \<CACHE_ID\>  [-t, -\-threshold] \<NUMBER\>
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/open_cas_linux/guide_tool_details.md]: Fixed in commit ae8644e

File: pages/release/open_cas_linux_release_notes.md [hunk 9]
```diff
@@ -11,3 +11,9 @@ Open CAS Linux initial release
 v19.06
 =======
 Added Write-Only mode
+
+v19.09
+======
+Added new IO classification rules  
+Updated CAS static calculations
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/release/open_cas_linux_release_notes.md]: Fixed in commit ae8644e

File: pages/release/open_cas_linux_release_notes.md [hunk 10]
```diff
@@ -11,3 +11,9 @@ Open CAS Linux initial release
 v19.06
 =======
 Added Write-Only mode
+
+v19.09
```
[the remark this answers is not in the document]
Comment (felipe-barajas) [reply pages/release/open_cas_linux_release_notes.md]: Fixed in commit ae8644e

**Comment by KamilLepek** [comment]
please resolve merge conflicts

**Pull request merged**